### ISSUE TITLE
Trainning loss around 0.08?

### ISSUE THREAD

**Issue by YuTingLiu** [YadiraF/PRNet#127]
Thanks for share this `unbelivable` projects.
We are family//
Since I decided to train my own model, uv map generating has been done, data comes form `300W-LP`, which contains 122450 samples. 
* What I am not sure is the trainning loss always around 0.08, output as follow:
![FireShot Capture 046 - TensorBoard - 192 168 5 109](https://user-images.githubusercontent.com/8286904/59477204-b9793100-8e86-11e9-9396-ce1e0da20ac6.png)
Can someone tell me how this happen?

**Comment by YuTingLiu** [comment]
Solved and closed, thanks.

**Comment by YuTingLiu** [comment]
![image](https://user-images.githubusercontent.com/8286904/60268648-748dd980-991f-11e9-9b64-d41994565219.png)
This is result after 3k iters, kinds of scary.
Anyway, the .obj file predicted by pre-trainned model all the same.

**Comment by HBLucasKo** [comment]
@YuTingLiu Hi!! Could you tell me how you solved the issue??

**Comment by qiannvMao** [comment]
@YuTingLiu Hi,as a newer to this, I failed to achieve the training code, could you please provide training code to me? I hope I can get your help, thank you very much. My email: mqn20170910@163.com

**Comment by michaelhuang74** [comment]
@YuTingLiu Hi Yuting, where did you get the training code or did you write the code by yourself? I also like to train the model using new datasets. Do you mind share your training code with me? Thanks a lot. My email is mqhuang@gmail.com

**Comment by JianBin-Liu** [comment]
> Thanks for share this `unbelivable` projects.
> We are family//
> Since I decided to train my own model, uv map generating has been done, data comes form `300W-LP`, which contains 122450 samples.
> 
> * What I am not sure is the trainning loss always around 0.08, output as follow:
>   ![FireShot Capture 046 - TensorBoard - 192 168 5 109](https://user-images.githubusercontent.com/8286904/59477204-b9793100-8e86-11e9-9396-ce1e0da20ac6.png)
>   Can someone tell me how this happen?
my trainning loss always around 2.00, did I need to train consistently?

**Comment by JianBin-Liu** [comment]
> Thanks for share this `unbelivable` projects.
> We are family//
> Since I decided to train my own model, uv map generating has been done, data comes form `300W-LP`, which contains 122450 samples.
> 
> * What I am not sure is the trainning loss always around 0.08, output as follow:
>   ![FireShot Capture 046 - TensorBoard - 192 168 5 109](https://user-images.githubusercontent.com/8286904/59477204-b9793100-8e86-11e9-9396-ce1e0da20ac6.png)
>   Can someone tell me how this happen?
my trainning loss always around 2.00, did I need to train consistently?
